subxt/src/client/online_client.rs

```diff
@@ -292,6 +295,11 @@ impl<T: Config> OnlineClient<T> {
         <Self as OfflineClientT<T>>::constants(self)
     }
 
+    /// Access custom types.
+    pub fn custom_types(&self) -> CustomTypes {
```

Comment:
It's really the values that we are accessing (and the values happen to come with type info) so maybe `custom_values -> CustomValues` makes more sense?